datalake/src/main/resources/duties/7.2.17/yarn/medium_duty_ha_scalable.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
This is more for completeness so that each shape is available on Yarn, but this is actually a duplicate of the `medium_duty_ha.json` and uses the same blueprint as normal HA, so this doesn't need to be tested.

Reply:
Let's YARN as well. I'm good with the current approach.